src/background.ts

```diff
@@ -5,19 +5,17 @@ const COLLABORATORS_SCOPE =
   'https://api.shopify.com/auth/partners.collaborator-relationships.readonly';
 let shopifyEmployee = false;
 
-function getOauth2Client(origin: string) {
+async function getOauth2Client(origin: string) {
+  const renderBackend = await getRenderBackend();
```

Comment:
On line 122 you call `getOauth2Client()` and `getRenderBackend()`, and then `getRenderBackend()` gets called again here. Is this an expensive operation? Can you see an easy way of eliminating the duplicate call?